### DISCUSSION TITLE
有没有关闭更新的配置

### DISCUSSION THREAD

**Discussion by FikaCode** [shuax/MouseInc.Settings#94] [Q&A]
因为2.11.4 和 2.11.5 会被公司的杀毒干掉，而2.11.3可以使用，但是每次开机都会提示更新，有没有一个设置可以去掉更新的提示呢？

**Comment by Caodaping** [comment]
我也遇到相似情况。我是2.11.4正常，但是更新到2.11.5被自动删除。我添加到了“排除列表”，依然被清理了，很奇怪。

**Comment by shuax** [comment]
杀毒软件的问题自己处理吧
